docs/release/release_0_4_3.md

```diff
@@ -0,0 +1,120 @@
+# napari 0.4.3
+
+We're happy to announce the release of napari 0.4.3!
+napari is a fast, interactive, multi-dimensional image viewer for Python.
+It's designed for browsing, annotating, and analyzing large multi-dimensional
+images. It's built on top of Qt (for the GUI), vispy (for performant GPU-based
+rendering), and the scientific Python stack (numpy, scipy).
+
+
+For more information, examples, and documentation, please visit our website:
+https://github.com/napari/napari
+
+## Highlights
+This release is the first release to have direct support for generating widgets from functions using [magicgui](https://napari.org/magicgui/) and the `viewer.window.add_functio_widget` method (#1856). We leverage the newly release `0.2` series of magicgui which seperates out an abstract function and widget API from its Qt backend. 
+
+In this release we also seperate out more of the Qt functionality from napari making it easier to run headless (#2039, #2055). We also add a `napari.run` method to then launch the napari application (#2056).
```

Comment:
Do you want to add a link to your animation repo here? It would be nice to explain *why* headless is even a desirable thing.

Reply:
I also feel like the two sentences make it feel like one follows from the other.  The ability to create a viewer without Qt installed, and the ability to start the Qt event loop with `napari.run` shouldn't come off as connected.  Maybe something like "we add a `napari.run` method as an alternative to using the `gui_qt` context manager?

Reply:
I've reworded to include what @tlambert03 suggests to make them appear less linked.

I don't want to link to napari-animation yet, it's massively in flux and not yet on pypi. That should be a goal for `0.4.4` - it will also only really work after we get the pydantic type validation stuff in as it makes assumptions on things being typed correctly.